<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopUpWindow06</class>
 <widget class="QDialog" name="PopUpWindow06">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>540</height>
    </rect>
   </property>
   <widget class="QLabel" name="line_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>120</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>120</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>120</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>120</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>120</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>120</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>120</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>120</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3S">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5S">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6S">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10S">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15S">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>15S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_30S">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>30S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_60S">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>60S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_120S">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>120S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_300S">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>300S</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
